03_launch_mgmt_cluster.sh

```diff
@@ -225,25 +184,18 @@ EOF
   else
     # Deploy Ironic using deploy.sh script
 
-    # Update container images to use local ones
-    cp "${BMOPATH}/ironic-deployment/ironic/ironic.yaml" "${BMOPATH}/ironic-deployment/ironic/ironic.yaml.orig"
-    cp "${BMOPATH}/ironic-deployment/keepalived/keepalived_patch.yaml" "${BMOPATH}/ironic-deployment/keepalived/keepalived_patch.yaml.orig"
-    update_kustomization_images "${BMOPATH}/ironic-deployment/ironic/ironic.yaml"
-    update_kustomization_images "${BMOPATH}/ironic-deployment/keepalived/keepalived_patch.yaml"
+    update_component_image Ironic "${IRONIC_LOCAL_IMAGE:-$IRONIC_IMAGE}"
+    update_component_image Mariadb "${MARIADB_LOCAL_IMAGE:-$MARIADB_IMAGE}"
+    update_component_image Keepalived "${IRONIC_KEEPALIVED_LOCAL_IMAGE:-$IRONIC_KEEPALIVED_IMAGE}"
+    update_component_image IPA-downloader "${IPA_DOWNLOADER_LOCAL_IMAGE:-$IPA_DOWNLOADER_IMAGE}"
 
     # Copy the generated configmap for ironic deployment
     cp "$IRONIC_DATA_DIR/ironic_bmo_configmap.env"  "${BMOPATH}/ironic-deployment/keepalived/ironic_bmo_configmap.env"
 
     # Deploy. Args: <deploy-BMO> <deploy-Ironic> <deploy-TLS> <deploy-Basic-Auth> <deploy-Keepalived>
     "${BMOPATH}/tools/deploy.sh" false true "${IRONIC_TLS_SETUP}" "${IRONIC_BASIC_AUTH}" true
-
-    # Restore original files
```

Comment:
We are no longer restoring original files ?
is it so that re-running deployment script will override it anyway or there is another step to do the restoration.

Reply:
Yes, I removed this because of 2 reasons:
1. In [03_launch_mgmt_cluster.sh we remove and clone again](https://github.com/metal3-io/metal3-dev-env/blob/master/03_launch_mgmt_cluster.sh#L38-L40) every time unless `FORCE_REPO_UPDATE` is set to false (default is true). So the cleaning will happen anyway the next time we run it.
2. I think it is easier to understand what is happening if we don't restore it immediately. It was very confusing to me at first that I could check in the manifests and see one thing and then in the cluster it would be something different. Now we will be able to see what modifications were made more easily. (And it is easy to restore manually also with `git reset --hard`.)

Maybe a third reason would also be that we sometimes need these changes also after the script is done. For example when pivoting. It becomes hard then to understand what is going on when the files are changed in the script and then restored in the playbook.

Reply:
Got it, thanks.